docs/guides/migrating-react-router-app.md

```diff
@@ -60,28 +60,32 @@ export default function handleRequest(
   request: Request,
   responseStatusCode: number,
   responseHeaders: Headers,
-  remixContext: EntryContext
+  remixContext: EntryContext,
+  loadContext: AppLoadContext
 ) {
   return isbot(request.headers.get("user-agent"))
     ? handleBotRequest(
         request,
         responseStatusCode,
         responseHeaders,
-        remixContext
+        remixContext,
+        loadContext
       )
     : handleBrowserRequest(
         request,
         responseStatusCode,
         responseHeaders,
-        remixContext
+        remixContext,
+        loadContext
       );
 }
 
 function handleBotRequest(
   request: Request,
   responseStatusCode: number,
   responseHeaders: Headers,
-  remixContext: EntryContext
+  remixContext: EntryContext,
+  _loadContext: AppLoadContext
```

Comment:
I don't think these need the underscore since they're not defined inside of `handleRequest`, so they're not in the same scope as the other `loadContext`?

Reply:
@brophdawg11 We are passing `loadContext` from `handleRequest` to `handleBotRequest` & `handleBrowserRequest` with this PR, but it's not used in `handleBotRequest` & `handleBrowserRequest`, hence the underscore

Reply:
I don't know that we need to bother passing it through to `handleBotRequest`/`handleBrowserRequest` then?  And I don't know that our templates should need to prefix the variable if it's unused.  The intention of having it there is so users can use if if they need/want to - and it feels wrong for them to have to change the name when they do so - or impart a naming preference on their newly created app?

Can we just simplify al templates to to the following and avoid touching the usility funcitons if they don't ever need to touch the contedxt?

```jsx
export default function handleRequest(
  request: Request,
  responseStatusCode: number,
  responseHeaders: Headers,
  remixContext: EntryContext,
  loadContext: AppLoadContext
) {
  return isbot(request.headers.get("user-agent"))
    ? handleBotRequest(
        request,
        responseStatusCode,
        responseHeaders,
        remixContext
      )
    : handleBrowserRequest(
        request,
        responseStatusCode,
        responseHeaders,
        remixContext
      );
}
```

Reply:
Should be as requested now 👍